Signal/src/util/Pastelog.m

```diff
@@ -150,6 +151,8 @@ - (void)uploadFileWithUploadUrl:(NSString *)uploadUrl fields:(NSDictionary *)fie
                 NSString *fieldValue = fields[fieldName];
                 [formData appendPartWithFormData:[fieldValue dataUsingEncoding:NSUTF8StringEncoding] name:fieldName];
             }
+            [formData appendPartWithFormData:[OWSMimeTypeApplicationZip dataUsingEncoding:NSUTF8StringEncoding] name:@"content-type"];
```

Comment:
This should be `self.mimeType`, not `OWSMimeTypeApplicationZip`.

Reply:
or probably `weakSelf.mimeType`.

Reply:
Done, thanks!